.github/workflows/deploy.yml

```diff
@@ -22,12 +22,17 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v2
 
+    - name: Set up QEMU for Arm64 containers
+      uses: docker/setup-qemu-action@v3
```

Comment:
Do you know that `ubuntu-24.04-arm` also exists? See https://github.com/actions/partner-runner-images, it's available for OSS by default.